lua/lualine/highlight.lua

```diff
@@ -1,24 +1,23 @@
 -- Copyright (c) 2020-2021 hoob3rt
 -- MIT license, see LICENSE for more details.
 local M = {}
-local utils_colors = require 'lualine.utils.cterm_colors'
+local utils_colors
```

Comment:
I would name this `cterm_colors` since that's all it is really

Reply:
I agree . Renamed